### ISSUE TITLE
 Ability for admins/other dashboard users to change the notification email sent from name

### ISSUE THREAD

**Issue by lexbi** [roundhouse/FormBuilder-2-Craft-CMS#51]
Hi,

I believe this is a feature request, but if it's possible another way please enlighten me!

I would like the ability for a craft user to update the form's name, for example, it may make more sense in the backend for a form to be called "Services Sidebar Contact Form", but, doing so would also set that as the sender name in notification emails to the submitter (if using notify submitter). 

I would like to have a form name that makes sense to the dashboard user, but, another that can be altered to make sense for the public too.

tl;dr - I would like a new field for the public form name.

![image](https://cloud.githubusercontent.com/assets/2337910/15816603/67ae52ea-2bcb-11e6-9108-4c5a678eaed6.png)

**Comment by owldesign** [comment]
I have pushed latest with new field added to the notifications toggle called Public Form Name. Let me know if this works for you and I'll close this ticket out.

Thanks.
![screen shot 2016-06-06 at 10 17 20 am](https://cloud.githubusercontent.com/assets/438764/15830964/e94cc4ae-2bcf-11e6-89aa-5308ac621a1f.png)

**Comment by lexbi** [comment]
Hi Vadim,

That's absolutely perfect, thanks for your help.
